nixos/modules/services/video/frigate.nix

```diff
@@ -578,11 +583,17 @@ in
       ] ++ optionals (!stdenv.hostPlatform.isAarch64) [
         # not available on aarch64-linux
         intel-gpu-tools
+        rocmPackages.rocminfo
       ];
       serviceConfig = {
-        ExecStartPre = pkgs.writeShellScript "frigate-clear-cache" ''
-          rm --recursive --force /var/cache/frigate/*
-        '';
+        ExecStartPre = [
```

Comment:
Why not use [systemd.services.<name>.preStart](https://search.nixos.org/options?channel=unstable&from=0&size=50&sort=relevance&type=packages&query=preStart) ?

Reply:
It was first contributed that way, and I like that I can create multiple named scripts with a clear purpose.